cilium-health/launch/endpoint.go

```diff
@@ -87,28 +85,30 @@ func configureHealthRouting(netns, dev string, addressing *models.NodeAddressing
 		routes = append(routes, v6Routes...)
 	}
 
-	prog := "ip"
-	args := []string{"netns", "exec", netns, "bash", "-c"}
+	prog := "bash"
+	args := []string{"-c"}
 	routeCmds := []string{}
 	for _, rt := range routes {
```

Comment:
Yep, I recall briefly wondering why this wasn't done before. I can't see any reason not to.

Reply:
I've improved it a bit,  in that we exec individually and I ditched some of the munging. It's still wrapped in `bash -c` because `ToIPCommand()` doesn't really return a proper `Command` style format and I don't wanna get too far afield with changes, but could drop that later.